scripts/set_core_version.py

```diff
@@ -60,6 +61,29 @@ def update_package_json(relpath, newversion):
         f.write("\n")
 
 
+def update_release_rs():
+    out = (
+        "// Generated by scripts/set_core_version.py.\n"
+        "//! DC release info.\n"
+        "\n"
+        "use chrono::NaiveDate;\n"
+        "use once_cell::sync::Lazy;\n"
+        "\n"
+        "/// Last release date.\n"
+    )
+    now = datetime.datetime.utcnow()
```

Comment:
Looks like we either should use the date from CHANGELOG.md or add the current date to CHANGELOG.md. What do u think?

Reply:
Good idea! But what do u think about a source of the date? I see that now we start to add it to CHANGELOG.md. Should i take it from there or better to fixup CHANGELOG.md from this python code?